app/routes/api/jobs.py

```diff
@@ -3,20 +3,23 @@
 """
 
 import os
-
+import os.path
```

Comment:
If we're importing `os`, I don't think we need to import `os.path`. 

`os.path` will be accessible from importing `os`